pages/api/register.js

```diff
@@ -8,10 +9,15 @@ export default async function handler (request, response) {
   // Process a POST request
   const newUser = await prisma.user.create({
     data: {
-      companyId: 2,
       email: request.body.email,
       name: request.body.fullName,
-      password: await bcrypt.hashSync(request.body.password)
+      password: bcrypt.hashSync(request.body.password)
```

Comment:
is this a `sync` function and the `await` is actually not needed?

Reply:
Yes. EsLint suggested that the await is not needed here.